notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit ('.venv')",
+   "metadata": {
+    "interpreter": {
+     "hash": "74f679a4fc854d75b381693882d369b7543a476b4b9cfda76eebb965e9131c3c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -q timesketch_api_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install ../api_client/python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timesketch_api_client import search\n",
+    "from picatrix.lib import state as state_lib\n",
+    "\n",
+    "import altair as alt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from timesketch_api_client import config\n",
+    "from timesketch_import_client import helper\n",
+    "from timesketch_import_client import importer\n",
+    "from timesketch_api_client import search\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "source": [
+    "# Create the sketch and import some data"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client = config.get_client(confirm_choices=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client.create_sketch(name=\"test\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sketch = ts_client.get_sketch(1)"
```

Comment:
ok, so first of all you have 

```python
ts_client.create_sketch(name='test')
```

above... which creates and returns a sketch... the problem here is a) you can't be sure that this newly created sketch is the first sketch ever created in someones installation, so this is not a guarantee of getting the new sketch... BUT since the `create_sketch` already returns the newly created sketch a much better approach is

```python
sketch = ts_client.create_sketch(name='test')
```

that way you create it,b ut also ensure that you get the correct sketch back to work with moving forward.

Reply:
good point